addons/sale_timesheet/static/tests/sale_timesheet_test_helpers.js

```diff
@@ -0,0 +1,15 @@
+import { AccountAnalyticLine } from "./mock_server/mock_models/account_analytic_line";
+import { ProjectTask } from "./mock_server/mock_models/project_task";
+import { SaleOrderLine } from "./mock_server/mock_models/sale_order_line";
+import { mailModels } from "@mail/../tests/mail_test_helpers";
+import { defineModels } from "@web/../tests/web_test_helpers";
+
+export const saleTimesheetModels = {
+    AccountAnalyticLine,
+    ProjectTask,
+    SaleOrderLine,
+};
```

Comment:
models should be defined in their main module otherwise we might run into some strange dependency issues if current module is not installed